### PR TITLE
11.0 link documents

### PR DESCRIPTION
**Pull request by miguel-domatix**
- Added helpdesk settings.
- Added a new menu to access the settings.
- Added two fields in settings to enable or disable this option and select the models to relate.
- Added new field in the ticket to relate this with other documents.

### CODE REVIEW

File: helpdesk_mgmt_link_documents/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+    'name': 'Helpdesk linker',
+    'summary': 'Allow link any document to helpdesk tickets',
+    'version': '11.0.1.8.0',
```
Comment (emagdalenaC2i) [inline helpdesk_mgmt_link_documents/__manifest__.py]:
```suggestion
    'version': '11.0.1.0.0',
```

File: helpdesk_mgmt_link_documents/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
+{
+    'name': 'Helpdesk linker',
```
Comment (emagdalenaC2i) [inline helpdesk_mgmt_link_documents/__manifest__.py]:
```suggestion
    'name': 'Helpdesk link documents',
```

File: helpdesk_mgmt_link_documents/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+    'summary': 'Allow link any document to helpdesk tickets',
+    'version': '11.0.1.8.0',
+    'license': 'AGPL-3',
+    'author': 'Domatix',
+    'website': 'https://www.domatix.com/',
```
Comment (emagdalenaC2i) [inline helpdesk_mgmt_link_documents/__manifest__.py]:
```suggestion
    'website': 'https://github.com/OCA/helpdesk',
```

**Review by emagdalenaC2i — CHANGES_REQUESTED** [review]
Please add the new README by fragment system copying https://github.com/OCA/maintainer-tools/tree/master/template/module/readme and editing or removing the needed sections

**Comment by pedrobaeza** [comment]
Please don't use plurals in module name: https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#modules

And I don't get why you finally did a module, as I said that it's OK to add a reference field in existing module.

**Comment by miguel-domatix** [comment]
I have made a new module for this:
https://github.com/OCA/helpdesk/pull/79#issuecomment-528285794

**Comment by pedrobaeza** [comment]
Well, I pointed several possible future problems. Any way, I see you have change the plural in the manifest, not the technical name (that's to say, the folder name).

**Comment by nacho-domatix** [comment]
> Well, I pointed several possible future problems.

This feature might be interesting for many companies. We can add it as an option (so can be disabled by UI) or as it is now (inheritance module).

**Comment by pedrobaeza** [comment]
I'm comfortable with both. What do you prefer?

**Comment by nacho-domatix** [comment]
UI option indeed, it's more "Odoo" this way.

**Pull request closed without merging**